tests/MiniCover.HitServices.UnitTests/HitTestMethodTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NFluent;
```

Comment:
Can we stick only with Shouldly?
As far as I know is the most popular assertion library for .NET: https://github.com/shouldly/shouldly

Reply:
Or, if NFluent have more advantages, can you completely replace shouldly by NFluent?